src/foam/u2/FloatView.js

```diff
@@ -92,16 +92,31 @@ foam.CLASS({
 
     function formatNumber(val) {
       val = (val || 0).toFixed(this.precision);
-      return this.trimZeros ? Number(val).toString() : val;
+      
+      // Add comma formatting
+      let parts = val.split('.');
+      parts[0] = parts[0].replace(/\B(?=(\d{3})+(?!\d))/g, ',');
```

Comment:
This should be done using the precision support and locale.   . and , in numbers are language dependent.
At least create another ticket to follow up with proper support.

Reply:
OK if you need it immediately for reflow.  
Hitherto, all the precision and ., formatting been focused on Currency.   
But with reflow, looks like we need to consider other numeric output - which could be a can of worms - hence another ticket.